react/src/pages/utils/components/BooleanDisplay.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from "react";
+import { PseudoBoolean, PseudoBooleanReadableMap } from "../typings";
+
+/**
+ * Displays a PseudoBoolean value in a user-readable manner.
+ * Specifically used for the render prop for Material-Table columns.
+ */
+export default function BooleanDisplay(props: {
+    value: any | PseudoBoolean;
+    fieldName: any;
+    type: "row" | "group";
+}) {
+    const [value, setValue] = useState<string>("");
+
+    useEffect(() => {
+        let switchValue: PseudoBoolean;
```

Comment:
Maybe this is better?
```suggestion
       const switchValue: PseudoBoolean = props.type === "row" ? (props.value as any)[props.fieldName] : props.value
```